corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js

```diff
@@ -191,30 +191,35 @@ hqDefine("enterprise/js/enterprise_dashboard", [
     }
 
     $(function () {
-        const datePicker = tempusDominus.createDefaultDateRangePicker(
-            document.getElementById("id_date_range"),
-            moment().subtract(30, "days"),
-            moment()
-        );
-
-        const formSubmissionsDisplay = MobileFormSubmissionsTile(datePicker);
-        const maxDateRangeDays = initialPageData.get("max_date_range_days");
-        const dateRangeModal = DateRangeModal(datePicker, dateRangePresetOptions, maxDateRangeDays, formSubmissionsDisplay);
-
-        $("#dateRangeDisplay").koApplyBindings(formSubmissionsDisplay);
-        $("#enterpriseFormsDaterange").koApplyBindings(
-            dateRangeModal
-        );
-
-        kissmetrics.track.event("[Enterprise Dashboard] Visited page");
+        const metricType = initialPageData.get('metric_type');
+        const $dateRangeDisplay = $("#dateRangeDisplay");
+        let dateRangeModal = null;
+        if ($dateRangeDisplay.length) {
+            const datePicker = tempusDominus.createDefaultDateRangePicker(
+                document.getElementById("id_date_range"),
+                moment().subtract(30, "days"),
+                moment()
+            );
+
+            const formSubmissionsDisplay = MobileFormSubmissionsTile(datePicker);
+            const maxDateRangeDays = initialPageData.get("max_date_range_days");
+            dateRangeModal = DateRangeModal(datePicker, dateRangePresetOptions, maxDateRangeDays, formSubmissionsDisplay);
+
+            $("#dateRangeDisplay").koApplyBindings(formSubmissionsDisplay);
```

Comment:
```suggestion
            $dateRangeDisplay.koApplyBindings(formSubmissionsDisplay);
```

Reply:
Addressed in 9519faf554874e82b03f4b024a883dc74d27cc77